src/osd/PrimaryLogPG.cc

```diff
@@ -54,6 +54,8 @@
 #include "include/assert.h"  // json_spirit clobbers it
 #include "include/rados/rados_types.hpp"
 
+#include <regex>
```

Comment:
please use `osd: ` as the prefix of the commit message's title. no need to include "src".